dace/symbolic.py

```diff
@@ -516,6 +516,7 @@ def sympy_intdiv_fix(expr):
     b = sympy.Wild('b', properties=[lambda k: k.is_Symbol or k.is_Integer])
     c = sympy.Wild('c')
     d = sympy.Wild('d')
+    e = sympy.Wild('e', properties=[lambda k: isinstance(k, sympy.Basic)])
```

Comment:
Even integers are sympy.Basic (`isinstance(sp.sympify('1'), sp.Basic)` --> `True`). Would this always work?

Reply:
Has this been resolved? We can probably add a test. If it is an issue, then I suggest to either do `isinstance(k, sympy.Expr)` or perhaps even better `isinstance(k, sympy.Basic) and not isinstance(k, sympy.Atom)`. 
From the SymPy documentation about the `Atom` class:
> A parent class for atomic things. An atom is an expression with no subexpressions.
>
> Examples
> 
> Symbol, Number, Rational, Integer, … But not: Add, Mul, Pow, …

Reply:
It should be solved now, I've explicitly added to not match Atom expressions